src/Timer.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { TimerProps, TimerState } from './models/models';
+
+const START_TIME = 100;
+
+class Timer extends React.Component<TimerProps, TimerState> {
+
+  state = { time: { minutes: 1, seconds: 0 }, songCount: 1};
+
+  constructor(props: TimerProps) {
+    super(props);
+    this.startTimer = this.startTimer.bind(this);
+  }
+
+  componentDidMount() {
+    this.startTimer();
+  }
+
+  async startTimer() {
+    // Loop through a list of [0,1,2,3...START_TIME-1]
+    for (const num of Array(START_TIME).keys()) {
```

Comment:
`Array(n)` will first create an array of length n. Then the `.keys()` method returns an Iterator of the keys, which for arrays, would correspond to 0,1,2,3...n. This is literally just a shorthand nifty way to get a list that contains [0,1....n]